### DISCUSSION TITLE
List Potential Grants for DaBible Foundation

### DISCUSSION THREAD

**Discussion by olasubomi-o** [DaBible-Foundation/DaBible-MainHub#186] [Ideas]
If you find any grants we can apply to be sure to comment on this issue.

- [x] Kingdom Mission Fund
- [x] Maclellan 
- [x] Amos Foundation 
- [x] https://www.grantrequest.com/AccountManager.aspx?sid=5514 
- [x] http://crowelltrust.org/apply-for-a-grant/ 
- [x] NCF- Contacted

If you find any grants we can apply to be sure to comment on this issue.

_Originally posted by @olasubomi-o in https://github.com/DaBible-Foundation/DaBible-MainHub/issues/101#issuecomment-848467528_

**Comment by sanmiajanaku** [comment]
I will be visiting the [Gideons Bible](https://gideons.org/) in Tennesse, or find a good contact point. Gideon's bible distributed the small blue bibles all across Africa.

**Comment by emmanlex** [comment]
I suggest we look into [Charity Water](charitywater.org) and their approach to transparency.

**Comment by apostle101** [comment]
I will look into the websites of the most trusted NGOs and how they convey their message to attain such a status
